templates/beta/contracts/BetaTemplateBase.sol

```diff
@@ -42,43 +52,118 @@ contract BetaTemplateBase {
         appIds = _appIds;
     }
 
-    function createDAO(string name, MiniMeToken token, address[] holders, uint256[] stakes, uint256 _maxTokens) internal returns (Voting) {
+    function latestVersionAppBase(bytes32 appId) public view returns (address base) {
```

Comment:
I understand why this has been reordered to be here. But I wonder whether we should, in exceptions like this one, keep the main contract (even if internal functions) on top and then start with the rule. Actually `latestVersionAppBase` is only a public function for convenience reasons (as it could perfectly be internal and the frontend could calculate it or use another contract for the purpose.)

Reply:
`solium` throws only a warning, so putting it back to its original position.

Reply:
We can also ignore specific lines (or files) for solium: see [docs](http://solium.readthedocs.io/en/latest/user-guide.html#configuring-with-comments).

Reply:
Nice, added to remove the annoying warning too. Thanks!